web/themes/custom/votegov/src/sass/uswds-overrides/usa-breadcrumb.scss

```diff
@@ -10,3 +10,10 @@
     @include vote-link-dark;
   }
 }
+
+// targeting the div wrapping the breadcrumb block and add margin top to the following element
+.system-breadcrumb-block + * {
```

Comment:
Targeting the div that is wrapping the breadcrumb block.  The `*` was not working on the element since this div was wrapping it